docs/Feature_Implementation_Matrix.md

```diff
@@ -0,0 +1,172 @@
+|:---	                |   :---:           | :---:	                    |:---                                                     |
+| Log V1 	            |   GA              | ✓                         |                                                         |
+| Log V2 (Skylog) 	  |   In development  |                           | [#1674](https://github.com/google/trillian/issues/1674) |
+| Map                 |   Alpha	          |   	                      |                                                         |
+| Log-Backed-Map  	  |   In development  |   	                      |                                                         |
+
+### Trillian Log V1
+
+This is feature complete, and is actively used in production by multiple CT log operators, including Google.
+
+### Trillian Log V2 (Skylog)
+
+Skylog is an append-only log with considerably higher throughput and lower integration latency than the v1 log.
+It currently exists only in prototype form, but is being developed.
+
+### Trillian Map
+
+The Trillian Map is currently considered experimental code; while it's functional
+and could certainly be used to prototype systems, its use is nuanced and it is
+not currently recommended to deploy it into a production system.
+
+(Also see section below on Log-Backed-Map for future work to make the map
+easier to use correctly.)
+
+### Log-backed-map
+
+The log backed map is a framework for combining log(s) and map(s) to build
+systems that are able to represent the evolution and current "world state"
+of an ecosystem.
+
+It's currently in a design phase.
+
+
+## Concrete implementations
+
+This section lists the status of implementations for the _pluggable_ subsystems which Trillian supports.
+
+### Storage
+
+Trillian supports "pluggable" storage implementations for durable storage of the merkle tree data.
+The state and characteristics of these implementations are detailed below.
+
+#### V1 log storage
+
+The Log storage implementations supporting the original Trilian log.
+
+
+| Storage  	      | Status  | Deployed in prod  	| Notes                                                                       |
+|:---	            | :---:   | :---:	              |:---                                                                         |
+| Spanner  	      | GA	    | ✓                   | Google internal-only, see CloudSpanner for external use.                    |
+| CloudSpanner  	| Beta 	  |   	                | Google maintains continuous-integration environment based on CloudSpanner.  |
+| MySQL  	        | GA      | ✓           	      |                                                                             |
+| Postgres  	    | In dev. |   	                | [#1298](https://github.com/google/trillian/issues/1298)                     |
+
+##### Spanner
+This is a Google-internal implementation, and is used by all of Google's current Trillian deployments.
+
+##### CloudSpanner
```

Comment:
Might be worth rationalizing why this is in Beta. One reasonable suggestion is to raise & link to an Issue tracking blockers for this becoming GA.

Reply:
Added a link to a tracking issue.